### ISSUE TITLE
`YAMLView` causes traceback

### ISSUE THREAD

**Issue by jceb** [cuducos/yaml.nvim#19]
In a file containing multiple yaml documents, the following error is thrown every now and then when I run `:YAMLView`. I haven't figured out what exactly triggers the error. Moving around a few times in the file and invoking the command triggers it:

```
E5108: Error executing lua ...im/site/pack/packer/opt/yaml.nvim/lua/yaml_nvim/pair.lua:72: attempt to concatenate local 'index' (a nil value)
stack traceback:
        ...im/site/pack/packer/opt/yaml.nvim/lua/yaml_nvim/pair.lua:72: in function 'get_keys'
        ...im/site/pack/packer/opt/yaml.nvim/lua/yaml_nvim/pair.lua:87: in function 'parse'
        ...im/site/pack/packer/opt/yaml.nvim/lua/yaml_nvim/init.lua:84: in function 'func'
        ...im/site/pack/packer/opt/yaml.nvim/lua/yaml_nvim/init.lua:28: in function 'assure_yaml_filetype'
        ...im/site/pack/packer/opt/yaml.nvim/lua/yaml_nvim/init.lua:78: in function 'view'
        [string ":lua"]:1: in main chunk
```

**Comment by jceb** [comment]
Ha, I found a way to reproduce the issue:

```yaml
a: {}
b:
  - c: 1
```

When `:YAMLView` is invoked on line 2, `b:`, the error is thrown.

**Comment by cuducos** [comment]
I added a hot-fix ( dc4f5d7 ), but maybe something useful for another issue is to have a conversation about how we expect the node `b` to be visualized, right?
